src/LIBMONGOCRYPT_VERSION_CURRENT

```diff
@@ -1 +1 @@
-1.0.4
+1.1.0
```

Comment:
As a heads up, the latest release is 1.2.0. The only notable change is the additional support for a `sessionToken` argument to allow authentication with temporary AWS credentials (https://jira.mongodb.org/browse/MONGOCRYPT-302).